<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormActOperate</class>
 <widget class="QWidget" name="FormActOperate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>815</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>815</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>815</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>-1</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">用户变量运算</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 28px &quot;微软雅黑&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="labelParaTitle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#labelParaTitle
{
	color: black;
	border-image: url(:/img/widget/Panel.png) 0 4 0 4;
	border-left: 4px transparent;
	border-right: 4px transparent;
}
</string>
     </property>
     <property name="text">
      <string>用户变量运算指令参数</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameActOperate">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="IsMainArea" stdset="0">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelVar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>用户变量：</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_Var</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="xLineEdit" name="lineEdit_Var">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelValue">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>操作数：</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_Var</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="xSpinBox" name="BoxValue">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="specialValueText">
         <string notr="true"/>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string notr="true"/>
        </property>
        <property name="prefix">
         <string notr="true"/>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>6000000</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="HasBtn" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBoxOperate">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="title">
         <string>运算符</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="horizontalSpacing">
          <number>4</number>
         </property>
         <property name="verticalSpacing">
          <number>2</number>
         </property>
         <property name="margin">
          <number>4</number>
         </property>
         <item row="0" column="0">
          <widget class="xRadioButton" name="radioPlus">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string notr="true"/>
           </property>
           <property name="statusTip">
            <string notr="true"/>
           </property>
           <property name="whatsThis">
            <string notr="true"/>
           </property>
           <property name="accessibleName">
            <string notr="true"/>
           </property>
           <property name="accessibleDescription">
            <string notr="true"/>
           </property>
           <property name="text">
            <string notr="true">＋</string>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="xRadioButton" name="radioSub">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string notr="true"/>
           </property>
           <property name="statusTip">
            <string notr="true"/>
           </property>
           <property name="whatsThis">
            <string notr="true"/>
           </property>
           <property name="accessibleName">
            <string notr="true"/>
           </property>
           <property name="accessibleDescription">
            <string notr="true"/>
           </property>
           <property name="text">
            <string notr="true">－</string>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="xRadioButton" name="radioMul">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string notr="true"/>
           </property>
           <property name="statusTip">
            <string notr="true"/>
           </property>
           <property name="whatsThis">
            <string notr="true"/>
           </property>
           <property name="accessibleName">
            <string notr="true"/>
           </property>
           <property name="accessibleDescription">
            <string notr="true"/>
           </property>
           <property name="text">
            <string notr="true">×</string>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="xRadioButton" name="radioDiv">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string notr="true"/>
           </property>
           <property name="statusTip">
            <string notr="true"/>
           </property>
           <property name="whatsThis">
            <string notr="true"/>
           </property>
           <property name="accessibleName">
            <string notr="true"/>
           </property>
           <property name="accessibleDescription">
            <string notr="true"/>
           </property>
           <property name="text">
            <string notr="true">÷</string>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="xRadioButton" name="radioMod">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string notr="true"/>
           </property>
           <property name="statusTip">
            <string notr="true"/>
           </property>
           <property name="whatsThis">
            <string notr="true"/>
           </property>
           <property name="accessibleName">
            <string notr="true"/>
           </property>
           <property name="accessibleDescription">
            <string notr="true"/>
           </property>
           <property name="text">
            <string notr="true">%</string>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="xRadioButton" name="radioAssign">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string notr="true"/>
           </property>
           <property name="statusTip">
            <string notr="true"/>
           </property>
           <property name="whatsThis">
            <string notr="true"/>
           </property>
           <property name="accessibleName">
            <string notr="true"/>
           </property>
           <property name="accessibleDescription">
            <string notr="true"/>
           </property>
           <property name="text">
            <string notr="true">=</string>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>xSpinBox</class>
   <extends>QSpinBox</extends>
   <header>xspinbox.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>xRadioButton</class>
   <extends>QRadioButton</extends>
   <header>xradiobutton.h</header>
  </customwidget>
  <customwidget>
   <class>xLineEdit</class>
   <extends>QLineEdit</extends>
   <header>xlineedit.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_Var</sender>
   <signal>clicked()</signal>
   <receiver>FormActOperate</receiver>
   <slot>selectVar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BoxValue</sender>
   <signal>clicked()</signal>
   <receiver>FormActOperate</receiver>
   <slot>inputValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectVar()</slot>
  <slot>inputValue()</slot>
 </slots>
</ui>
